safekeeper/src/http/routes.rs

```diff
@@ -566,6 +602,10 @@ pub fn make_router(conf: SafeKeeperConf) -> RouterBuilder<hyper::Body, ApiError>
         .delete("/v1/tenant/:tenant_id", |r| {
             request_span(r, tenant_delete_handler)
         })
+        .get(
+            "/v1/tenant/:tenant_id/timeline/:timeline_id/snapshot",
+            |r| request_span(r, timeline_snapshot_handler),
+        )
         .post("/v1/pull_timeline", |r| {
             request_span(r, timeline_pull_handler)
         })
```

Comment:
nit: now we can also delete `"/v1/tenant/:tenant_id/timeline/:timeline_id/file/:filename",` handler two lines below

Reply:
done